packages/atomic/rollup.config.js

```diff
@@ -2,12 +2,21 @@ import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from 'rollup-plugin-typescript2';
 import replacePlugin from '@rollup/plugin-replace';
+import {readFileSync} from 'fs';
 
 const isProduction = process.env.BUILD === 'production';
 
+/**
+ * @returns {{ version: string }}
+ */
+function getPackageConfiguration() {
+  return JSON.parse(readFileSync('package.json', 'utf-8'));
```

Comment:
`package.json` and `lerna.json` don't always contain the same version. `package.json`'s version isn't bumped when there's no changes within its associated package.